backend/app/adapter/db/url.go

```diff
@@ -99,6 +99,10 @@ WHERE "%s"=$1;`,
 
 // GetByAliases finds URLs for a list of aliases
 func (u URLSql) GetByAliases(aliases []string) ([]entity.URL, error) {
+	if len(aliases) == 0 {
+		return []entity.URL{}, nil
+	}
+
```

Comment:
GetByAliases should not throw error when aliases is an empty slice

Reply:
I think we should do the same thing for the previous PR.